<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DronesWindow</class>
 <widget class="QWidget" name="DronesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kirk Project - Gestion des drones</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/files/Ressources/KirkProject.png</normaloff>:/files/Ressources/KirkProject.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QPushButton" name="pbSupprimer">
     <property name="toolTip">
      <string>Supprimer le drone sélectionné</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pbNouveau">
     <property name="toolTip">
      <string>Créer un nouveau drone</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pbSelectioner">
     <property name="text">
      <string>Sélectionner Drone</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QPushButton" name="pbAnnuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pbModifier">
     <property name="toolTip">
      <string>Modifier le drone sélectionné</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="4" colspan="2">
    <widget class="DroneDetails" name="detDrone" native="true"/>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QListWidget" name="list_drones"/>
   </item>
   <item row="4" column="2" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DroneDetails</class>
   <extends>QWidget</extends>
   <header>GestionDrones/DroneDetails.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
